### DISCUSSION TITLE
List organization members through Rest API

### DISCUSSION THREAD

**Discussion by riyasahal** [community/community#33533] [Apps, API and Webhooks]
I have been working on an integration, as a part of it I have to create a preset that pulls the user list of an organisation. The list does not contain emails of users. Is there a separate endpoint from where we could get this information or an app that can help us get some information about user emails belonging to an organisation?

Ref: https://docs.github.com/en/rest/orgs/members

**Comment by byrneh** [comment]
@riyasahal  you willneed to use the GraphQL to retrieve for all memebers
Additionally you can only view(retrieve) public emails or emails that match a verified domain on your your organization and/or enterprise account.

**Reply by riyasahal** [reply]
@byrneh 
So, if the domain has been verified, I will be able to fetch the email ids for all the users within an organisation? 
I see that enterprise edition supports the fetching of user email ids but in case of organisations, we do not have a separate end point for the same. That'll only allow me to fetch public emails, right?
